process/process.go

```diff
@@ -202,7 +204,10 @@ func (p *Process) Run() error {
 		if err != nil {
 			return err
 		}
-
+		err = p.postStart()
+		if err != nil {
+			p.logger.Error("[Process] postStart failed: %v", err)
+		}
```

Comment:
Minor style thing, if you're not keeping any return values, I believe this is the preferred form:

```suggestion
		if err := p.postStart(); err != nil {
			p.logger.Error("[Process] postStart failed: %v", err)
		}
```

It's less sensitive/leaky to existing/future values of `err`.